src/cms/utils/pdf_utils.py

```diff
@@ -82,13 +82,19 @@ def generate_pdf(region, language_slug, pages):
             title = region.name
     language = Language.objects.get(slug=language_slug)
     filename = f"Integreat - {language.translated_name} - {title}.pdf"
+    # django-webpack-loader assumes request object in context
+    # and that this request object has a specific attribute set
+    dummy_request = type(
+        "dummy_request", (object,), {"_webpack_loader_used_tags": set()}
+    )()
```

Comment:
`dummy_request = type("", (), {})()` should be enough, right? The attribute `_webpack_loader_used_tags` is set by webpack loader if not existent on the request object anyway...
Or, maybe even better: do an `from django.http import HttpRequest` and use `HttpRequest()` as dummy request? maybe a bit more readable...

Reply:
Ok, I took the real request for error templates and a dummy HttpRequest object for the PDF export